### ISSUE TITLE
Text selection not working

### ISSUE THREAD

**Issue by ankurpramanik** [stephanrauh/ngx-extended-pdf-viewer#116]
When a text is going to be selected. It is not working

**Comment by prashantguleria** [comment]
In documentation it's written If you set [handTool]="false" it should work but for me also text layer is not rendering.

**Comment by prashantguleria** [comment]
@ankurpramanik I figured it out.  In file ngx-extended-pdf-viewer.js @ line number 841 says         `options.set('textLayerMode', this.showHandToolButton ? 1 : 0);`

Just set the [showHandToolButton] = "true" and textLayer should come.

Just try this : 

`                <ngx-extended-pdf-viewer  [src]="pdfURL" [useBrowserLocale]="true" [handTool]="false" [showHandToolButton]="true" [contextMenuAllowed]="false" [showSidebarButton]="false" showPropertiesButton="false"> </ngx-extended-pdf-viewer>
`

**Comment by stephanrauh** [comment]
@ankurpramanik Thanks for reporting the bug. However, next time I'd appreciate some more detail. The text selection tool worked fine in my applications. It's a lucky circumstance that @prashantguleria stepped in. His first post added at least some detail. His second post allowed me to find and fix the bug in a few minutes.

**Comment by stephanrauh** [comment]
@prashantguleria Awesome analysis, thanks!

**Comment by stephanrauh** [comment]
I've just released version 1.0.0-rc.6. This should solve the bug.

Enjoy!
Stephan

**Comment by prashantguleria** [comment]
> @prashantguleria Awesome analysis, thanks!

You're welcome @stephanrauh :)

**Comment by ArinaWik** [comment]
Hey, 
I am facing the same issue. Text selection is not working at my application as well ( neither in Browser, nor in real device). 
I have tried the recommended solutions mentioned above, however no success.
Here is my code:   
<ngx-extended-pdf-viewer  [src]="src" [useBrowserLocale]="true" 
  [handTool] = "false"
  (handTool) = "false"
  [textLayer]="true"
  [showHandToolButton]="true" 
  [contextMenuAllowed]="false" 
  [showSidebarButton]="false"  
  showPropertiesButton="false"> 
</ngx-extended-pdf-viewer>

**VERSION** : "ngx-extended-pdf-viewer": "^5.0.0-alpha.5",

**Comment by stephanrauh** [comment]
@ArinaWik Is there a `<div>` called `text-layer`? If it's missing, is the text layer there when you open the file on https://mozilla.github.io/pdf.js/web/viewer.html?

If it's missing on the Mozilla site, there's nothing you can do about it. Some documents don't have text, but are all graphics.

**Comment by ArinaWik** [comment]
Hey, thanks for your very fast answer. I have tested on https://mozilla.github.io/pdf.js/web/viewer.html and yes there is text layer div. While checking it I have figured out that the problem is when I am going to chrome://inspect devices even in  https://mozilla.github.io/pdf.js/web/viewer.html text selection is not working. When I exit the chrome device inspection its working... Is the feature available for mobile devices? Might be there is some feature to be activated in order to work on real device and not only on browser? Or ? Thanks again a lot..
